### PR TITLE
Recruiter & Admin Roles

### PR DESCRIPTION
**Pull request by Aboudik1452**
### Pull Request for Recruiter & Administrator Roles

This pull request, referencing issue #106, is for adding the recruiter and administrator role. The following changes are:

- Added Select (drop down) feature to choose the account type.
- Modified user model, controller, and session storage to include role.
- Added redirect to the jobs page by clicking on the jobs button in the navigation bar.
- Added role handling for job posting pages.
- CSS changes to the sign up page.

Let me know if you have any comments or concerns! 🙂

### CODE REVIEW

File: src/client/src/components/SignUp.jsx [hunk 1]
```diff
@@ -119,6 +128,21 @@ function SignUp() {
...
+                    <br />
+                    <Select required id="select" value={userData.role} 
+                    sx={{width: 250, height: 40, backgroundColor: "#a3c5d0",borderColor: "white"}}
+                    onChange={(e) =>
+                      setUserData({ ...userData, role: e.target.value })
+                    }>
+                      <MenuItem value={"User"}>User</MenuItem>
+                      <MenuItem value={"Recruiter"}>Recruiter</MenuItem>
+                      <MenuItem value={"Administrator"}>Administrator</MenuItem>
+                      <MenuItem value={"Illegal"}>Illegal</MenuItem>
```
Comment (imran1289-ah) [inline src/client/src/components/SignUp.jsx:141]: What is this selection used for ?

**Review by imran1289-ah — APPROVED** [review]
Overall this is good !! I did had some comments so if you can clarify them that would be nice, Other than that good job!

File: src/client/src/components/SignUp.jsx (hunk 1 above)
Comment (Aboudik1452) [reply src/client/src/components/SignUp.jsx:141]: It's there to test the collection constraint. I'll remove it.

File: src/client/src/components/EditJobPosting.jsx [hunk 2]
```diff
@@ -38,80 +64,90 @@ function EditJobPosting() {
...
-    //Edit posting page
-    <Container id="editPostcontainer">
-      <Row>
-        <Col md={12}>
-          <div className="WrapperEditPost">
-            <h3 className="Title"> EDIT YOUR POSTING </h3>
-            <form onSubmit={savePost}>
-              <div className="FormEditPost">
-                <label className="PlaceholderEditPost">Title</label>
+  if (userID && userID && (userRole === "Recruiter" || userRole === 'Administrator')) {
```
[the remark this answers is not in the document]
Comment (Aboudik1452) [reply src/client/src/components/EditJobPosting.jsx:82]: Will remove it.

**Review by Kunal22shah — APPROVED** [review]

**Comment by Kunal22shah** [comment]
Looks good and works on my end!

**Pull request merged**